visualization/app/codeCharta/ui/resetSettingsButton/resetSettingsButton.component.html

```diff
@@ -1,4 +1,6 @@
 <button (click)="applyDefaultSettings()" [title]="tooltip">
-	<i class="fa fa-undo"></i>
+	<div class="icon-wrapper">
```

Comment:
[optional] do we need this wrapper or could we solve it through selectors of existing elements?

Reply:
You are right! we can get by without it.